composer.json

```diff
@@ -36,15 +36,20 @@
     "the-events-calendar/coding-standards": "dev-master",
     "the-events-calendar/tec-testing-facilities": "dev-master",
     "vlucas/phpdotenv": "^4.1",
-    "wp-coding-standards/wpcs": "^2.1"
+    "wp-coding-standards/wpcs": "^2.1",
+    "phpstan/phpstan": "^1.10",
+    "phpstan/phpstan-deprecation-rules": "^1.1",
+    "phpstan/extension-installer": "^1.3",
+    "szepeviktor/phpstan-wordpress": "*"
```

Comment:
This is in the hopes of never falling behind the current WP version.
See convo here: https://github.com/the-events-calendar/the-events-calendar/pull/4348#discussion_r1313505629